xwiki-platform-core/xwiki-platform-rendering/xwiki-platform-rendering-macros/xwiki-platform-rendering-macro-include/src/main/java/org/xwiki/rendering/internal/macro/include/IncludeMacro.java

```diff
@@ -202,7 +202,29 @@ public void setDocumentDisplayer(DocumentDisplayer documentDisplayer)
                 references.pop();
             }
         }
+        
+        // Exclude First Heading 
+        if (parameters.excludeFirstHeading()) {
+
+            // Getting the first block.
+            Block firstBlock = result.getChildren().get(0);
+
+            // Check if the first Block is Section block.
```

Comment:
Not needed, this says exactly the same thing as the code below. When you write comments, don't explain what you do but WHY you're doing it.